iocuddle/src/lib.rs

```diff
@@ -46,7 +46,7 @@
 //! Otherwise, classic `ioctl` interfaces can be defined and used via the
 //! `Ioctl::classic()` constructor as follows:
 //!
-//! ```
+//! ```no_run
```

Comment:
Hrm. I carefully designed this test to use device nodes that should be available everywhere. What causes this error?

Reply:
I actually can't reproduce this in a container or on my laptop, which is unsurprising because the device node is present in both. I must have gotten my wires crossed somewhere when I was playing around with my Enarx tree. I'll drop this patch.